### DISCUSSION TITLE
Ghost cursor mod

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#18701] [Ideas]
Idea: A mod that overlays a greyscale version of the cursor from the Auto replay into gameplay. I feel that this could be a very useful mod for people wanting to initially learn the timing of lower AR maps (especially with HD) since they can more easily gauge where their cursor is supposed to be at any given point in time without having to guess. I think that it would also be great to use when trying to learn how to hit low AR streams or spaced streams in general since you will be able to see how fast your cursor needs to be moving in order to hit them accurately. I could also see it being useful when playing tech maps or maps with cutstreams and awkward patterns.

**Comment by mk56-spn** [comment]
This might come with the eventual "practice mod" that has been conceptualised before, but id agree that this would be nice to have

**Comment by smoogipoo** [comment]
Related to https://github.com/ppy/osu/discussions/13119
